### ISSUE TITLE
A11Y: accessible testing scenarios

### ISSUE THREAD

**Issue by mahdikhashan** [boudicca-events/boudicca.events#336]
related issues:

1. #289 
2. #294 
3. #293 
  3.1 A11Y for blind users
4. #290 
  4.1 A11Y needs on accessing information (blind, ...)
  4.2 A11Y needs for the events (wheelchair, ...)

TODO:
- discuss test scenarios documentation ([WCAG2](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=131%2C134%2C127))
- investigate related testing tools
- communicate with the UX fellow

possible scenarios: 

1. Perceivability:
    - all not-textual content convey required information
  
2. keyboard navigation
3. screen readers
4. forms
5. buttons
6. interactive components

**Comment by mahdikhashan** [comment]
I manually tested using chrome screen reader:

what I noticed (possible issues):
1. header is not a list, results in mis-understanding for people using screen readers. (should check relevant WCAG guideline)
2. an event dom object is not focusable using tab, not sure what would be the case using an accessible navigator.
3. the Boudicca logo alt information is in German
4. the image of an event does not have any alt tag. result in mis-information.
5. when the filter drawer is opened, the screen reader losses it's focus when reaches the end of the elements in the drawer. in this case, continues with the event items in the lower layer. in this case the user can not close the drawer.

**Comment by mahdikhashan** [comment]
a [tool](https://accessibilityinsights.io/docs/web/overview/) to manually test the common a11y issues, suggested by Playwright.

**Comment by mahdikhashan** [comment]
> I manually tested using chrome screen reader:
> 
> what I noticed (possible issues):
> 
> 1. header is not a list, results in mis-understanding for people using screen readers. (should check relevant WCAG guideline)
> 2. an event dom object is not focusable using tab, not sure what would be the case using an accessible navigator.
> 3. the Boudicca logo alt information is in German
> 4. the image of an event does not have any alt tag. result in mis-information.
> 5. when the filter drawer is opened, the screen reader losses it's focus when reaches the end of the elements in the drawer. in this case, continues with the event items in the lower layer. in this case the user can not close the drawer.

we fixed all the issues mentioned here already.

**Comment by mahdikhashan** [comment]
It has been done in a pr called manual testing related to user tests, also I have generated a couple of user test scenarios which was added to the google drive folder under UI-UX research.
